roles/puppet_repositories/vars/main.yml

```diff
@@ -1,3 +1,5 @@
 foreman_puppet_repositories_supported_versions:
   - "5"
```

Comment:
This was dropped from yum.puppet.com so should this be dropped? Perhaps in a separate PR.

Reply:
Yeah.

Reply:
https://github.com/theforeman/foreman-operations-collection/pull/153